Gemfile

```diff
@@ -25,6 +25,7 @@ group :development, :test do
 	gem 'database_cleaner'
 	gem 'shoulda-matchers'
   gem 'launchy'
+  gem 'coveralls', require: false
```

Comment:
Try to fix the indentation.

Reply:
If it looks good in your ide like it did with antonella, delete both lines and write them again.